### PR TITLE
Update sys-dm-server-memory-dumps-transact-sql.md

### PR DESCRIPTION
**Pull request by DanielAdeniji**
Added  SQL Server 2008 R2 Service Pack 1 ( *SP1* ) to indicate when the dynamic manangement view ( sys.dm_server_memory_dumps ) was introduced.

### CODE REVIEW

**Comment by PRMerger18** [comment]
@DanielAdeniji : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/system-dynamic-management-views/sys-dm-server-memory-dumps-transact-sql.md [hunk 1]
```diff
@@ -23,7 +23,9 @@ ms.author: wiassaf
 # sys.dm_server_memory_dumps (Transact-SQL)
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
 
-  Returns one row for each memory dump file generated by the [!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]. Use this dynamic management view to troubleshoot potential issues.  
+The sys.dm_server_memory_dumps dynamic management view ( *DMV* ) was introduced in SQL Server 2008 R2 Service Pack 1 ( *SP1* ).
+
+Returns one row for each memory dump file generated by the [!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]. Use this dynamic management view to troubleshoot potential issues.
```
Comment (WilliamDAssafMSFT) [inline docs/relational-databases/system-dynamic-management-views/sys-dm-server-memory-dumps-transact-sql.md:28]:
```suggestion
Returns one row for each memory dump file generated by the [!INCLUDE[ssDEnoversion](../../includes/ssdenoversion-md.md)]. Use this dynamic management view to troubleshoot potential issues.  

Introduced in [!INCLUDE[sskilimanjaro-md](../../includes/sskilimanjaro-md.md)] Service Pack 1.
```

Thanks for the contribution! Updated styling to use include, standardize.

**Comment by PRMerger14** [comment]
@WilliamDAssafMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by WilliamDAssafMSFT** [comment]
Thanks @DanielAdeniji for submitting this PR. I've added a collaborating commit. Your commit will be merged and this article update is in flight. After the updated article is published, GitHub will list you as an article contributor. 

Thanks again for this feedback, your suggestions help improve our documentation.

#sign-off

**Pull request merged**